### DISCUSSION TITLE
music/rample/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/music#14] [General]
# music/rample/

https://shanenull.com/music/rample/

**Comment by shane0** [comment]
# rample

6–bit or 8–bit, 44100 Hz

folder name cap letter then number

A1 A2 etc

file name

1 2 3 4 .wav


## jamuary & other playlists

- 2024 https://www.youtube.com/playlist?list=PLGY2UhH7nNtJSN_2jR49Cokwdv4wW1YJ4
- 2023 https://youtube.com/playlist?list=PLGY2UhH7nNtKj3uGLluafQHE4_56sVdmU&si=si6w2NNrFKekCCZ2
- 2022 https://youtube.com/playlist?list=PLGY2UhH7nNtIGbIGwvO-XZGbKy6JCLUcs&si=c6Bmo-n1ozIpCjhN
- 2020 deity EP https://youtube.com/playlist?list=PLGY2UhH7nNtJDriq1EQe3GD8WqbKHUlhE&si=HqIX_y1RUqSXPxCA

## stuff

https://squarp.net/rample/manual/#Zj8++7M

**Reply by shane0** [reply]
- for now the hapax is setup to use track 1 midi a channel 16 4 notes below
- need to create a new instrument file looks like nobody shared one idk why squarp doesn't release an instrument file...
- need to update my default template
- the erica synth bassline module is using cv 1
- later I will decide if the rample can replace the blackbox
- the hydrasynth is also not on the desk atm
- midi was strange the midi cable they provide is way too short & did not work for me so I used the midi dongle
- things i will research is if this can multisample or if converting all my samples to rample spec is easily done

**Reply by shane0** [reply]
Squarp
instruments
HAPAX
L1
[2
C3
СЧ
С*Ч ОЧ
Ey
ПЧВ СНБ 049СИ16 П50СН16 П51С16
D7
СЧ
ПЧЕСНІБ
Da
D4
C#4
Е•Ч
h4g CH16
n50c16 n51CH16

**Comment by shane0** [comment]
![h](https://private-user-images.githubusercontent.com/1454458/374009330-e3501444-3469-4d39-b453-9a2cbb95827b.jpeg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjgyNjk4MzgsIm5iZiI6MTcyODI2OTUzOCwicGF0aCI6Ii8xNDU0NDU4LzM3NDAwOTMzMC1lMzUwMTQ0NC0zNDY5LTRkMzktYjQ1My05YTJjYmI5NTgyN2IuanBlZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMDclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDA3VDAyNTIxOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWM4MmZlYmYxMmE4NDVkZTVhYmM0MTUxMjFlMDcxYzA3ZGZmMjFkMTcyOGVlZDVhNWZkMjU2YmJlMDI2MWMzYWYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.FXdDn7z4exXdvCa8vzWE5v3XnWXtQew95MOnOELF-Ao)

**Reply by shane0** [reply]
- first attempt to use this with the hapax
- feels like something that could be integrated why am I having to manually setup midi between two products in the same system?
- 4 tracks but i have 8 mapped by default on hapax drums
- research if i can setup 8 samples without rigamarole
- see if i can convert samples to the brittle format specs and possibly eliminate the blackbox?

**Reply by shane0** [reply]
why is there no instrument file for this already do i have to craft one?

**Reply by shane0** [reply]
https://youtu.be/G8yh84aC8f4
First test

**Comment by shane0** [comment]
# 6 string bass

- sounds so nasty
- https://youtube.com/shorts/mlCIM9KzVVA?si=3DfshtEGCbIbEFR-
- https://www.instagram.com/reel/DBAmdT2JKGZ/?igsh=MXA4MjZ3Z2ZlcWdlNQ==

**Comment by shane0** [comment]
# chaconne

- The bass line in Bach's Chaconne is repeated 64 times.

**Comment by shane0** [comment]
Sax battle
https://youtu.be/RPjL909DiVI?si=P2pzStWbydoFYBHG

**Reply by shane0** [reply]
Coltrane

https://youtu.be/lJ7QTRzV9RM?si=5g3cNS-DWxn9_Owc
